src/CommunityToolkit.Aspire.Microsoft.Data.Sqlite/AspireSqliteExtensions.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Aspire;
+using HealthChecks.Sqlite;
+using Microsoft.Data.Sqlite;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Provides extension methods for registering Sqlite-related services in an <see cref="IHostApplicationBuilder"/>.
+/// </summary>
+public static class AspireSqliteExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:Sqlite:Client";
+
+    /// <summary>
+    /// Registers <see cref="SqliteConnection" /> as a singleton in the services provided by the <paramref name="builder"/>.
+    /// </summary>
+    /// <param name="builder">The <see cref="IHostApplicationBuilder" /> to read config from and add services to.</param>
+    /// <param name="name">The connection name to use to find a connection string.</param>
+    /// <param name="configureSettings">An optional method that can be used for customizing the <see cref="SqliteConnectionSettings"/>. It's invoked after the settings are read from the configuration.</param>
+    /// <remarks>Reads the configuration from "Aspire:Sqlite:Client" section.</remarks>
+    /// <exception cref="InvalidOperationException">If required ConnectionString is not provided in configuration section.</exception>
+    public static void AddSqliteConnection(
+        this IHostApplicationBuilder builder,
+        string name,
+        Action<SqliteConnectionSettings>? configureSettings = null) =>
+            AddSqliteClient(builder, DefaultConfigSectionName, configureSettings, name, serviceKey: null);
+
+    /// <summary>
+    /// Registers <see cref="SqliteConnection" /> as a keyed singleton for the given <paramref name="name" /> in the services provided by the <paramref name="builder"/>.
```

Comment:
```suggestion
    /// Registers <see cref="SqliteConnection" /> as keyed scoped for the given <paramref name="name" /> in the services provided by the <paramref name="builder"/>.
```

We register scoped not singleton.